sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -1855,16 +1885,28 @@ case class StringSpace(child: Expression)
     Examples:
       > SELECT _FUNC_('Spark SQL', 5);
        k SQL
+      > SELECT _FUNC_(encode('Spark SQL', 'utf-8'), 5);
```

Comment:
IMO the examples are becoming more complicated after changes... The example are duplicated and making it harder to read.
How about we have one or two examples for binary at the end with a comment?

Reply:
This PR follows up other string expressions with binary examples.
Of course, I have no objection to keeping only one or two examples for binary.

Reply:
Oops, thanks @gengliangwang for addressing this.